instrumentation/opentelemetry-instrumentation-celery/src/opentelemetry/instrumentation/celery/utils.py

```diff
@@ -12,15 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from __future__ import annotations
+
 import logging
-from typing import ContextManager, Optional, Tuple
+from typing import TYPE_CHECKING, Optional, Tuple
 
 from celery import registry  # pylint: disable=no-name-in-module
 from celery.app.task import Task
 
 from opentelemetry.semconv.trace import SpanAttributes
 from opentelemetry.trace import Span
 
+if TYPE_CHECKING:
+    from contextlib import AbstractContextManager
```

Comment:
It will work outside the `TYPE_CHECKING` anyway here, it was introduced on Python 3.6. But if you want to be explicit what is only for typing, it's okay.

Reply:
I want to be explicit that is only for typing. Thanks!!